src/services/notification.service.js

```diff
@@ -0,0 +1,28 @@
+import { authHeader } from '@/helpers';
+import { config } from "@/configurations";
+import axios from "axios";
+
+export const notificationService = { getNotification };
+
+function getNotification() {
```

Comment:
This function won't be required at all. This file can be deleted.

Reply:
Sir, it is being used.